setup.md

```diff
@@ -76,6 +94,18 @@ To install and run Flutter, your development environment must meet these minimum
    $ echo $PATH
    ```
 
+#### Windows
+
+1. Add Flutter to your PATH environment variable by opening 
+**Control Panel>System and Security>System>Advanced system settings>Environment Variables...**
+1. Modify the `PATH` system variable to add the full path to the `flutter\bin` directory.
+   For example, `C:\flutter\bin`.
+
+   If you don't have admin access to your machine, open **Control Panel>User Accounts>User Accounts>Change my environment variables**
+   and add a `PATH` user variable with the path to `flutter\bin` as its value. 
+
```

Comment:
Yes, John's change was just for `flutter doctor`, we still need this for `flutter run/build` until https://github.com/flutter/flutter/issues/8515 is fixed.